internal/unifi/client.go

```diff
@@ -25,14 +25,19 @@ type httpClient struct {
 }
 
 const (
-	unifiLoginPath  = "%s/api/auth/login"
-	unifiRecordPath = "%s/proxy/network/v2/api/site/%s/static-dns/%s"
+	unifiLoginPathUDM         = "%s/api/auth/login"
+	unifiLoginPathNetworkApp  = "%s/api/login"
+	unifiRecordPathUDM        = "%s/proxy/network/v2/api/site/%s/static-dns/%s"
+	unifiRecordPathNetworkApp = "%s/v2/api/site/%s/static-dns/%s"
```

Comment:
```suggestion
	unifiLoginPathUDM         = "%s/api/auth/login"
	unifiLoginPathNetworkApp  = "%s/api/login"
	unifiRecordPathUDM        = "%s/proxy/network/v2/api/site/%s/static-dns/%s"
	unifiRecordPathNetworkApp = "%s/v2/api/site/%s/static-dns/%s"
```

Can these vars match the names of the config option? e.g. `unifiLoginPathController` / `unifiLoginPathGateway` ?